openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/FeedRepository.java

```diff
@@ -935,6 +935,10 @@ private boolean fieldsChanged(Thread original, Thread updated) {
                 || !Objects.equals(
                     original.getAnnouncement().getEndTime(),
                     updated.getAnnouncement().getEndTime())))
+        || (original.getAi() == null && updated.getAi() != null)
```

Comment:
why is this specific Ai() is necessary

Reply:
here I'm storing the last updated query from the conversation with the MetaPilot to easily pass the history to the API

Reply:
should also help users go over the thread history but have a quick look at what was the outcome

Reply:
if the thread belongs to a bot in this case it would be Metapilot, do we need this specific condition of getAi() type? and also it should be getAI() as its abbreviation